build/helpers/moment.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var moment = require('moment');
+
+module.exports = function (context, options) {
+	var date = options ? context : new Date();
+	var options = options || context;
+	var format = options.hash.format || 'YYYY-MM-DD ';
+	// http://momentjs.com/docs/#/displaying/
+	return moment(date).utc().format(format);
```

Comment:
I thought this was something we ended up requiring historically because of timezone issue. Admittedly, I copied this part blindly from @tylersticka's example. I'll look into it.

Should this be an optional thing?